src/backends/sentry_backend_crashpad.cpp

```diff
@@ -78,31 +78,36 @@ typedef struct {
     sentry_path_t *breadcrumb1_path;
     sentry_path_t *breadcrumb2_path;
     size_t num_breadcrumbs;
+    sentry_value_t event_id;
 } crashpad_state_t;
 
 static void
-sentry__crashpad_backend_user_consent_changed(sentry_backend_t *backend)
+crashpad_backend_user_consent_changed(sentry_backend_t *backend)
```

Comment:
is there a particular reason for renaming all these functions?

Reply:
The main reason is that across the modules, it seemed that the naming scheme is (or was at some point):

* `sentry_*` for all publicly available interfaces via `sentry.h`
* `sentry__*` for all private interfaces which cross module boundaries (i.e., are only internally exposed via module headers)
* no prefix for all static (i.e., module-private) functions, which shouldn't easily conflict in any "namespace"

So one part is trivial consistency, but it also gets easier to navigate if I know from the name if there might be an external contract to consider.